qutip/qobj.py

```diff
@@ -1577,21 +1596,29 @@ def matrix_element(self, bra, ket):
 
         Parameters
         -----------
-        bra : :class:`qutip.Qobj`
-            Quantum object of type 'bra' or 'ket'
+        bra : :obj:`.Qobj`
+            Quantum object of type :obj:`.bra`
```

Comment:
don't link `bra`, and you've also changed the description.  It's allowed to pass a "ket" in this argument.